styles.css

```diff
@@ -2,12 +2,589 @@ body,
+  background: #dedee6;
+}
+
+.img-before-get-started {
+  display: none;
+}
+
+.img-before-get-started_desktop {
+  grid-column: 1;
+  grid-row: 1 / 3;
+  width: 100%;
+  height: auto;
+  object-fit: fill;
+}
+
+.image-after-started-image {
+  grid-column: 3;
+  grid-row: 3 / 5;
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+  align-self: end;
+  justify-self: start;
+}
+
+.get-started-section {
+  grid-column: 2;
+  grid-row: 1 / 5;
+  margin-bottom: 50vh;
+}
+
+.get-started-background > section > h1 {
+  transform: none;
+}
+
+.get-started-background > .get-started-section > .section-title {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 80px;
+  line-height: 100%;
+  text-align: center;
+}
+
+.get-started-background > .get-started-section > .section-title > .greetings {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  text-align: center;
+}
+
+.form-message {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: auto;
+}
+
+.yours {
+  border: none;
+}
+
+.yours label {
+  display: block;
+  width: 80%;
+  color: #8f8c8c;
+  background: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 13px;
+  line-height: 15px;
+  padding-left: 5%;
+  padding-right: 5%;
+  padding-top: 5px;
+  margin-left: 5%;
+  margin-right: 5%;
+  border-left: 1px solid rgb(173, 171, 171);
+  border-right: 1px solid rgb(173, 171, 171);
+  border-bottom: none;
+  border-top: 1px solid rgb(173, 171, 171);
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+}
+
+.yours input,
+.yours textarea {
+  display: block;
+  width: 80%;
+  background: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 15px;
+  padding-left: 5%;
+  padding-right: 5%;
+  margin-left: 5%;
+  margin-right: 5%;
+  border-left: 1px solid rgb(173, 171, 171);
+  border-right: 1px solid rgb(173, 171, 171);
+  border-top: none;
+  border-bottom: 1px solid rgb(173, 171, 171);
+  border-bottom-left-radius: 5px;
+  border-bottom-right-radius: 5px;
+}
+
+.yours textarea {
+  height: 200px;
+}
+
+/* start about section desktop version */
+
+/* .all-about {
+  display: grid;
+  grid-template-columns: 55% 45%;
+  grid-template-rows: auto;
+}
+
+.works-lfs-grid {
+  grid-column: 1;
+  grid-row: 1;
+  display: grid;
+  grid-template-columns: 5% 1fr 1fr 3%%;
+  grid-template-rows: auto;
+  color: #fff;
+}
+
+.languages-section {
+  background: #1c528a;
+  grid-column: 2;
+  grid-row: 1;
+}
+
+.frameworks-section {
+  background: #e2505c;
+  grid-column: 3;
+  grid-row: 1;
+}
+
+.skills-section {
+  background: #febf52;
+  grid-column: 2;
+  grid-row: 2;
+}
+
+.about-section {
+  grid-column: 2;
+  grid-row: 1;
 } */
 
-@media only screen and (min-width: 375px) {
+/* end of  about section
+
+/* mobile version */
+@media screen and (max-device-width: 767px),
```

Comment:
- [ ] Please note that we are building based on a mobile-first approach. Your media query should be based on min-width and not max-width. `@media screen and (min-width: value px) { }` 🪖

Reply:
noted, will take much time to fixes, otherwise i notice and learn from this, should be mobile-first approach not desktop-first approach.